std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -142,6 +142,20 @@ public static Column fromItems(String name, List<Value> items) {
     return new Column(name, storage);
   }
 
+  /** Creates a column from an Enso array. No polyglot conversion happens. This is unsafe */
+  public static Column fromItemsRaw(String name, List<Object> items, StorageType expectedType) throws ClassCastException {
```

Comment:
I do remember when `fromItems` introduced `List<Value>`... deciding whether `List<Value>` is needed or whether `List<Object>` is enough is a delicate dance balancing GraalVM Host Interop, GraalVM version, Enso implementation of interop and needs of the libraries.

In any case, I believe the #1 thing we need is to get stable, independent _benchmarks being executed daily by the CI_. Without it we will just shift the code back and forth without being sure the system is improved.

Reply:
For now I will keep it as is, but I will be adding tickets to revisit this.